dev/io.openliberty.org.jboss.resteasy.validator.provider/resources/META-INF/services/javax.validation.spi.ValidationProvider

```diff
@@ -0,0 +1 @@
+org.hibernate.validator.HibernateValidator
```

Comment:
I wonder if we need this file at all.  I think that when the `beanValidation-3.0` feature is enabled, that will automatically supply a ValidationProvider for us.